release-notes/24.05/index.html

```diff
@@ -0,0 +1,80 @@
+<li>To strengthen library compartmentalization, we have completed an adjustment to the ABI for functions whose arguments do not fit in registers.  These arguments are now reachable via a bounded capability passed in register C9 and newly compiled callees access them via C9 allowing callers to place argument values anywhere they desire.  This is compatible with code compiled for 23.11, which used a transitional ABI setting C9 in the caller, but not using it in the callee.  Code compiled for releases prior to 23.11 is no longer supported.
+<li>User level heap temporal safety performance has been improved by adding in-kernel worker threads.  This change allows userspace threads to invoke the kernel revoker, proceed with work, and then come back later to complete revocation.
+<li>Our CHERI-aware port of the bhyve hypervisor has been synced with AArch64 support merged to upstream FreeBSD.  The GDB stub has been enhanced to support reading capabilities in guest memory.
+<li>Kernels may now be compiled with the pure-capability benchmark ABI in addition to the existing hybrid and pure-capability ABIs. This ABI uses a modified form of code generation which improves performance on the Arm Morello board, required due to limitations with bounds prediction in the current Morello prototype. The performance of the benchmark ABI is more predictive of potential future CHERI microarchitectural performance than the Morello prototype running software compiled for CheriABI, making it preferable for benchmarking. However, this comes at the cost of reduced security, and so software compiled for the benchmark ABI should not be used for security evaluation. For details on how to use the benchmark ABI, and benchmark Morello in general, readers should see the Getting Started Guide.
+<li>The <tt>MAP_CHERI_NOSETBOUNDS</tt> flag to the mmap system call is now deprecated and a no-op.  When mmap is called with the <tt>MAP_FIXED</tt> flag and a pointer to an existing reservation, the passed pointer will be returned unmodified on success.
+</ul>
+<p>
+While CheriBSD is, by definition, an experimental research operating system, with the exception of the benchmark ABI, the above features are not yet considered to be mature. 
+</p>
+<p>
+As with previous releases, the default CheriBSD kernel on Morello ships with debugging features enabled, which should be disabled by booting a non-debug kernel before any performance benchmarking is performed.
+</p>
+<h2>Known Issues</h2>
+<ul>
+<li>ZFS support is lightly tested. The on-disk format is believed to be stable, but it is not impossible that future releases will be unable to read pools created by this release, particularly pools created on pure-capability kernels.
```

Comment:
but it is not impossible -. but it is possible that

Reply:
This is just copied verbatim from 23.11, which itself copied it verbatim from 22.12

Reply:
It's always nice to improve our writing over time ;-)  Even if it was my own mistake.

Reply:
I debated if we should leave this, but it's probably still accurate.